src/components/MailSession.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
```

Comment:
Desde o React 17, não precisamos importar o `React` por conta do "novo" [JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).


```suggestion
import { useState } from 'react';
```